test/Mustache/Test/TokenizerTest.php

```diff
@@ -354,10 +350,10 @@ public function getTokens()
 
     /**
      * @dataProvider getUnclosedTags
-     * @expectedException Mustache_Exception_SyntaxException
      */
     public function testUnclosedTagsThrowExceptions($text)
     {
+	    $this->expectException(Mustache_Exception_SyntaxException::class);
```

Comment:
There's various indenting issues like this that should be fixed

Reply:
Good catch, fixed now.